packages/assets-controllers/src/TokenRatesController.ts

```diff
@@ -120,14 +124,18 @@ enum PollState {
  * @type TokenRatesState
  *
  * Token rates controller state
- * @property contractExchangeRates - Hash of token contract addresses to exchange rates
+ * @property contractExchangeRates - Hash of token contract addresses to exchange rates keyed by chain ID and native currency (ticker)
  * @property supportedChains - Cached chain data
```

Comment:
looks like this doesn't exist in state anymore?

Reply:
Looks like its a class variable and shouldn't be listed as part of state anymore...

Reply:
done https://github.com/MetaMask/core/pull/2015/commits/1e77d0144008a44e537808ec5547a4dbb512dab4